azure-kusto-ingest/azure/kusto/ingest/_kusto_ingest_client.py

```diff
@@ -27,69 +29,128 @@ def __init__(self, kcsb):
         """
         kusto_client = KustoClient(kcsb)
         self._resource_manager = _ResourceManager(kusto_client)
+    
+    def ingest_from_dataframe(self, df, ingestion_properties):
+        blobs = []
 
-    def ingest_from_multiple_files(self, files, delete_sources_on_success, ingestion_properties):
+        file_name = 'df_{timestamp}_{pid}.csv.gz'.format(timestamp=int(time.time()), pid=os.getpid())
+        temp_file_path = os.path.join(tempfile.gettempdir(), file_name)
+
+        df.to_csv(temp_file_path, index=False, encoding='utf-8', header=False, compression='gzip')
```

Comment:
Can we write directly to blob, instead of going via the file?

Reply:
...and if we can't, why do you upload the file to blob instead of callin ingest_from_files?